src/pcap/cmd/pcap-api/main.go

```diff
@@ -48,7 +48,18 @@ func main() {
 		log.Fatal("unable to create api", zap.Error(err))
 	}
 
-	api.RegisterResolver(&pcap.BoshHandler{Config: config.ManualEndpoints})
+	// set up a BoshResolver for each bosh environment
+	for _, env := range config.BoshEnvironments {
+		resolver, err := pcap.NewBoshResolver(env, 8083) // FIXME
```

Comment:
The port should be part of the BOSH configuration, which contains a list of `Environment`s.